personal_assistant/contacts/templates/contacts/index.html

```diff
@@ -0,0 +1,30 @@
+{% extends "pa_core/home.html" %} {% block title%}
+    Contacts
+{% endblock %}
+
+{% block content %}
+
+    <nav class="nav">
+        <ul>
+            <li><a href="{% url 'contacts:add_contact' %}" role="button" class="btn btn-success">Create contact</a></li>
+        </ul>
+    </nav>
```

Comment:
Без обгортки наче краще виглядає:
```suggestion
    <a href="{% url 'pa_contact:create' %}" role="button" class="btn btn-success">Create contact</a>
```